bundles/org.openhab.binding.openweathermap/src/main/resources/ESH-INF/thing/channel-types.xml

```diff
@@ -105,6 +105,156 @@
+			<channel id="temperature" typeId="system.outdoor-temperature"/>
+			<channel id="apparent-temperature" typeId="apparent-temperature"/>
+			<channel id="pressure" typeId="system.barometric-pressure"/>
+			<channel id="humidity" typeId="system.atmospheric-humidity"/>
+			<channel id="dew-point" typeId="dew-point"/>
+			<channel id="uvindex" typeId="uvindex"/>
+			<channel id="wind-speed" typeId="system.wind-speed"/>
+			<channel id="wind-direction" typeId="system.wind-direction"/>
+			<channel id="gust-speed" typeId="gust-speed"/>
+			<channel id="cloudiness" typeId="cloudiness"/>
+			<channel id="rain" typeId="rain"/>
+			<channel id="snow" typeId="snow"/>
+			<channel id="visibility" typeId="visibility"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="oneCallMinutely">
+		<label>One Call API Minutely Forecast</label>
+		<description>Minutely precipitation delivered by the One Call API.</description>
+		<channels>
+			<channel id="time-stamp" typeId="time-stamp"/>
+			<channel id="precipitation" typeId="precipitation"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="oneCallHourly">
+		<label>One Call API Hourly Forecast</label>
+		<description>Hourly weather forecast delivered by the One Call API.</description>
+		<channels>
+			<channel id="time-stamp" typeId="hourly-forecast-time-stamp"/>
+			<channel id="condition" typeId="condition"/>
+			<channel id="condition-id" typeId="condition-id"/>
+			<channel id="icon" typeId="condition-icon"/>
+			<channel id="icon-id" typeId="condition-icon-id"/>
+			<channel id="temperature" typeId="system.outdoor-temperature"/>
+			<channel id="apparent-temperature" typeId="apparent-temperature"/>
+			<channel id="pressure" typeId="system.barometric-pressure"/>
+			<channel id="humidity" typeId="system.atmospheric-humidity"/>
+			<channel id="dew-point" typeId="dew-point"/>
+			<channel id="wind-speed" typeId="system.wind-speed"/>
+			<channel id="wind-direction" typeId="system.wind-direction"/>
+			<channel id="gust-speed" typeId="gust-speed"/>
+			<channel id="cloudiness" typeId="cloudiness"/>
+			<channel id="precip-probability" typeId="precip-probability"/>
+			<channel id="rain" typeId="rain"/>
+			<channel id="snow" typeId="snow"/>
+			<channel id="visibility" typeId="visibility"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="oneCallDaily">
+		<label>One Call API Daily Forecast</label>
+		<description>Daily weather forecast delivered by the One Call API.</description>
+		<channels>
+			<channel id="time-stamp" typeId="daily-forecast-time-stamp"/>
+			<channel id="sunrise" typeId="sunrise"/>
+			<channel id="sunset" typeId="sunset"/>
+			<channel id="condition" typeId="condition"/>
+			<channel id="condition-id" typeId="condition-id"/>
+			<channel id="icon" typeId="condition-icon"/>
+			<channel id="icon-id" typeId="condition-icon-id"/>
+			<channel id="temperature" typeId="system.outdoor-temperature"/>
```

Comment:
This channel should not be available in the daily forecast

Reply:
You are right, thanks. Removing it.